server/src/main/java/org/apache/druid/segment/loading/SegmentLoaderLocalCacheManager.java

```diff
@@ -44,7 +44,7 @@
 {
   private static final EmittingLogger log = new EmittingLogger(SegmentLoaderLocalCacheManager.class);
   private static final Comparator<StorageLocation> COMPARATOR = (left, right) ->
-      Longs.compare(right.available(), left.available());
+      Longs.compare(right.availableSizeBytes(), left.availableSizeBytes());
```

Comment:
`Comparator.comparingLong(StorageLocation::availableSizeBytes).reversed()` would be clearer

Reply:
Done.